### ISSUE TITLE
Move user profile from community page to users page

### ISSUE THREAD

**Issue by Joehuu** [ppy/osu-web#3095]
Since there is a users page now, the user profile should be there. Right now it is on the community page.
https://osu.ppy.sh/users/2/modding
https://osu.ppy.sh/users/2

**Comment by peppy** [comment]
I don't understand what you are saying.

**Comment by Joehuu** [comment]
The links I referenced end with `/users`. The user profile though is under the "community"(i.e. red violet header, community underlined). The modding page is on the right place ("users" with the light blue header).

I'm wondering if the [user profile](https://osu.ppy.sh/users/2) should be on "users"(i.e. the one with the light blue header).

I got "users or user" from the middle title of the tab `user modding history · user | osu!` which imo really needs to be changed to like `peppy's modding history | osu!` similar to profile title `peppy's profile | osu!`.
